lib/craft.rb

```diff
@@ -0,0 +1,7 @@
+class Craft
+  attr_reader :name, :supplies_required
+  def initialize(name, supplies_required)
+    @name = name
+    @supplies_required = supplies_required
+  end
+end 
```

Comment:
This looks great. Nice job setting up the object to initialize with the correct attributes.